10-JavaScript/fun-with-functions.md

```diff
@@ -42,6 +42,19 @@ sayItTwice("JS is OK!"); // "JS is OK!"
                          // "JS is OK!"
 ```
 
+##### Default Function arguments
+
+Sometimes, you may want to provide a default value in case a user does not specify an argument.  Just with with Ruby, JavaScript functions can take default Arguments.  You specify the default value for a parameter by setting the variable equal to a value in the function header's parentheses.
```

Comment:
"Just with with Ruby" -> "Just as with Ruby". We should probably also change "parameter" in the last sentence to "argument" to keep it consistent with the rest of this section.